src/Microsoft.ML.AutoML/API/ExperimentBase.cs

```diff
@@ -102,16 +102,17 @@ public abstract class ExperimentBase<TMetrics, TExperimentSettings>
             const int crossValRowCountThreshold = 15000;
 
             var rowCount = DatasetDimensionsUtil.CountRows(trainData, crossValRowCountThreshold);
+            bool isNotImageClassificationTask = columnInformation.ImagePathColumnNames.Count != 1;
```

Comment:
nit: isImageClassificationTask may be a simpler name?

Reply:
or hasImagePathColumns. because I suppose a classification task could have image path columns